src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -53,6 +53,44 @@ import './Menu.css'
   {{/menu-content}}
 {{/menu}}
 ```
+### Danger menu item
+```hbs
+{{#> menu}}
+  {{#> menu-content}}
+    {{#> menu-list}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Action 1
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> screen-reader}}Danger Item{{/screen-reader}}
```

Comment:
case and a colon - not sure if it matters, but that's what we use in the alert, modal, and popover.

```suggestion
            {{#> screen-reader}}Danger item:{{/screen-reader}}
```

Reply:
Applied update related to this edit, thanks